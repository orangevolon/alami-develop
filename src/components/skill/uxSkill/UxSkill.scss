@import "../../../assets/styles/badge.scss";

.ux-skill__article {
  .badges__div {
    height: 270px;
  }

  .photoshop-badge__img {
    @include badge(0.6);

    left: 50%;
    top: 0;
  }

  .illustrator-badge__img {
    @include badge(0.8);

    left: 35%;
    top: 36%;
  }

  .sketch-badge__img {
    @include badge(1.2);

    top: 40%;
    left: 75%;
  }

  .xd-badge__img {
    @include badge(0.85);

    top: 77%;
    left: 58%;
  }
}
