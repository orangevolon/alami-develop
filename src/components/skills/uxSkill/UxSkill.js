import { Badge, Skill } from "@components";
import photoshopBadge from "@assets/badges/photoshop.svg";
import illustratorBadge from "@assets/badges/illustrator.svg";
import sketchBadge from "@assets/badges/sketch.svg";
import xdBadge from "@assets/badges/xd.svg";

import "./UxSkill.scss";

const UxSkill = () => {
  const photoshopBadgeElement = Badge({ src: photoshopBadge });
  photoshopBadgeElement.classList.add("photoshop-badge__img");

  const illustratorBadgeElement = Badge({ src: illustratorBadge });
  illustratorBadgeElement.classList.add("illustrator-badge__img");

  const sketchBadgeElement = Badge({ src: sketchBadge });
  sketchBadgeElement.classList.add("sketch-badge__img");

  const xdBadgeElement = Badge({ src: xdBadge });
  xdBadgeElement.classList.add("xd-badge__img");

  const skillElement = Skill({
    badges: [
      photoshopBadgeElement,
      illustratorBadgeElement,
      sketchBadgeElement,
      xdBadgeElement
    ],
    title: "UI/UX",
    description: `
    Even though, I’m not a UI/UX designer now, I have a robust understanding of the user experience concepts, from color psychology to the methodologies of the user research. I’ve also mastered several design tools including Sketch, Xd, Illustrator and Photoshop over the course of the last decade.
    `
  });
  skillElement.classList.add("ux-skill__article");

  return skillElement;
};

export default UxSkill;
