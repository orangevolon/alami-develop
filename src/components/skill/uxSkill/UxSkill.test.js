import UxSkill from "./UxSkill";

describe("Testing components/skill/uxSkill", () => {
  it("Should render the component correctly", () => {
    expect(UxSkill()).toMatchSnapshot();
  });
});
