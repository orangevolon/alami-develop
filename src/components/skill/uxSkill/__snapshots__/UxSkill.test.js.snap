// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing components/skill/uxSkill Should render the component correctly 1`] = `
<article
  class="skill__article ux-skill__article"
>
  <h3
    class="title__h3"
  />
  <p
    class="description__p"
  />
  <div
    class="badges__div"
  >
    <img
      class="badge__img photoshop-badge__img"
      src="[object Object]"
    />
    <img
      class="badge__img illustrator-badge__img"
      src="[object Object]"
    />
    <img
      class="badge__img sketch-badge__img"
      src="[object Object]"
    />
    <img
      class="badge__img xd-badge__img"
      src="[object Object]"
    />
  </div>
</article>
`;
